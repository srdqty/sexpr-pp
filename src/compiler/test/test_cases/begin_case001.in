(begin 1)
(begin 1 2)
(begin 1 2 3)
