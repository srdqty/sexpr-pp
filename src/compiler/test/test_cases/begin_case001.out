(Begin (Constant 1))
(Begin (Constant 1) (Constant 2))
(Begin (Constant 1) (Constant 2) (Constant 3))
